tests/unit/bias/metrics/test_basic_stats.py

```diff
@@ -35,3 +35,105 @@ def test_confusion_matrix():
     assert [TP, FP, FN, TN] == basic_stats.confusion_matrix(
         dfB[0], sensitive_facet_index, dfB_pos_label_idx, dfB_pos_pred_label_idx
     )
+
+
+def test_observed_label_distribution():
+    sensitive_facet_index = dfB[0] == "F"
+    label_dist = [approx(3 / 7.0), approx(4 / 7.0)]
+    assert label_dist == basic_stats.observed_label_distribution(dfB[0], sensitive_facet_index, dfB_pos_label_idx)
+
+    sensitive_facet_index = dfB[0] == "M"
+    label_dist = [approx(2 / 5.0), approx(3 / 5.0)]
+    assert label_dist == basic_stats.observed_label_distribution(dfB[0], sensitive_facet_index, dfB_pos_label_idx)
+
+
+def test_performance_metrics():
```

Comment:
Could we also add a test which computes these metrics directly from `dfBinary` and `dfMulticategory`?

Reply:
Per offline discussion, I need to test calculating right number of TP/TN/FP/FN from df directly. So I moved it to a helper function